### ISSUE TITLE
It doesn't seen to work

### ISSUE THREAD

**Issue by Joaquin-123** [MrDiamond64/betterdiscord-showhiddenchannels#1]
Hi, I tried to uuse this version, but it seems to not do anything, maybe I have to do something more apart from changing the plug-in?

**Comment by XxUnkn0wnxX** [comment]
Because the author nuked the original plugin he even went out of his way to sabotage it, as well as force an auto-update if it detects a plugin named "ShowHiddenChannels.plugin.js"

These are the only two that I found that are working
https://github.com/maromalo/DefinitelyNotSHC
https://github.com/Saalvage/BetterDiscordAddons/tree/master/Plugins/ShowHiddenChannelsV2
https://github.com/NotCapengeR/ShowHiddenChannelsReturns - not working atm author says he will fix soon..

these at least don't require a custom version of `0BDFDB` Libary or a custom `0BDFDB.data` file..
however, with future updates to BD & the library, this plugin may have to be re-written from the ground up not dependent on DevilBro's Library file.

other than that may have to switch to powercord or something similar where guidelines are more lax..

**Comment by Joaquin-123** [comment]
Thanks, I tried to use the salvage version, tho it seems to not work, maybe 'cause I first tried replacing the code to the original file, I will try getting the plugin directly

**Comment by XxUnkn0wnxX** [comment]
you have to save it as `ShowHiddenChannelsV2.plugin.js` for salvage's version.

**Comment by Joaquin-123** [comment]
Still nothing, I think I have to change something else maybe

**Comment by Joaquin-123** [comment]
Ok, reintsalling the additional library seemed to help

**Comment by angrybirdice** [comment]
the first link worked but second one doesn't work

**Comment by angrybirdice** [comment]
any other discontinued plugins?

**Comment by Squishyzbear** [comment]
> Ok, reintsalling the additional library seemed to help

reinstalling devilbros library?, what additional library

**Comment by Joaquin-123** [comment]
Yep, I was refering to that one, tho it seems that it doesn't work anymore, so I installed this one instead 
https://github.com/JustOptimize/return-seeHiddenChannels

**Comment by Squishyzbear** [comment]
> Yep, I was refering to that one, tho it seems that it doesn't work anymore, so I installed this one instead https://github.com/JustOptimize/return-seeHiddenChannels

do i rename the plugin or does their liibrary need it named the same

**Comment by Joaquin-123** [comment]
Oh, this one uses it's own fork of the library, so you just replace then on the plug-in folder

**Comment by mimsy55** [comment]
> Because the author nuked the original plugin he even went out of his way to sabotage it, as well as force an auto-update if it detects a plugin named "ShowHiddenChannels.plugin.js"
> 
> These are the only two that I found that are working https://github.com/maromalo/DefinitelyNotSHC https://github.com/Saalvage/BetterDiscordAddons/tree/master/Plugins/ShowHiddenChannelsV2 https://github.com/NotCapengeR/ShowHiddenChannelsReturns - not working atm author says he will fix soon..
> 
> these at least don't require a custom version of `0BDFDB` Libary or a custom `0BDFDB.data` file.. however, with future updates to BD & the library, this plugin may have to be re-written from the ground up not dependent on DevilBro's Library file.
> 
> other than that may have to switch to powercord or something similar where guidelines are more lax..

the owner if this is a dick tbh  good thing theres ways around his bullcrap
